components/quick_input/quick_input.test.tsx

```diff
@@ -4,7 +4,7 @@
 import React from 'react';
 import {mount} from 'enzyme';
 
-import QuickInput from 'components/quick_input/index';
+import {QuickInput} from './quick_input';
```

Comment:
Question : should we prefer named exports over defaults

Reply:
I think we should still generally prefer the default export. The reason I'm doing the named export here is to bypass the `forwardRef` so that the snapshot still shows the body of the component